pkg/app/api/service/webservice/service.proto

```diff
@@ -401,3 +405,25 @@ message ListAPIKeysRequest {
 message ListAPIKeysResponse {
     repeated model.APIKey keys = 1;
 }
+
+message GetApplicationDeploymentFrequencyRequest {
```

Comment:
Still is it needed? I feel like all operations can be done via GetInsightDataRequest.

Reply:
Oh, I forgot removing :+1: